features/facebookImport/src/model/entities/connected-advertiser.js

```diff
@@ -0,0 +1,40 @@
+export default class ConnectedAdvertiser {
+    constructor() {
+        this._name = null;
+        this._hasDataFileCustomAudience = false;
+        this._hasRemarketingCustomAudience = false;
+        this._hasInPersonStoreVisit = false;
+    }
+
+    get name() {
+        return this._name;
+    }
+
+    set name(name) {
+        this._name = name;
+    }
+
+    get hasDataFileCustomAudience() {
+        return this._hasDataFileCustomAudience;
+    }
+
+    set hasDataFileCustomAudience(booleanValue) {
+        this._hasDataFileCustomAudience = booleanValue;
+    }
+
+    get hasRemarketingCustomAudience() {
+        return this._hasRemarketingCustomAudience;
```

Comment:
I wonder if it's worth the while to mark them as "private" (with the underscore) and then put getters for them anyway... Same above.

Reply:
It could be still useful to indicat the case when something should not be set. (Just when adding serialisation support we need to set all attributes)